libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -724,22 +743,32 @@ namespace SohImGui {
                 }
 
                 EnhancementCheckbox("No Clip", "gNoClip");
+		Tooltip("Allows you to walk through walls");
                 EnhancementCheckbox("Climb Everything", "gClimbEverything");
+		Tooltip("Makes every surface in the game climbable");
                 EnhancementCheckbox("Moon Jump on L", "gMoonJumpOnL");
+		Tooltip("Holding L makes you float into the air");
                 EnhancementCheckbox("Super Tunic", "gSuperTunic");
+		Tooltip("Makes every tunic have the effects of every other tunic");
                 EnhancementCheckbox("Easy ISG", "gEzISG");
+		Tooltip("Automatically activates the Infinite Sword glitch, making you constantly swing your sword");
                 EnhancementCheckbox("Unrestricted Items", "gNoRestrictItems");
+		Tooltip("Allows you to use all items at any age");
                 EnhancementCheckbox("Freeze Time", "gFreezeTime");
+		Tooltip("Freezes the time of day");
 
                 ImGui::EndMenu();
             }
 
             if (ImGui::BeginMenu("Developer Tools"))
             {
                 EnhancementCheckbox("OoT Debug Mode", "gDebugEnabled");
+		Tooltip("Enables Debug Mode, allowing you to select maps with L + R + Z, noclip with L + Dpad Right,\n and open the debug menu with L on the pause screen");
```

Comment:
Over in #316 I standardized on using "D-pad" as that seems to be the closest thing to an "official" Nintendo formatting. My PR sucks; if you're so inclined you might just want to merge those changes into yours since it's already in the same area and mine can be closed.